packages/gauges/src/constants/config/prod.ts

```diff
@@ -203,7 +203,7 @@ export const CONFIG_PROD: GaugeConfig[] = [
     type: GaugeType.V3,
     token0Address: bscTokens.ankrETH.address,
     token1Address: bscTokens.eth.address,
-    feeTier: FeeAmount.LOW,
+    feeTier: FeeAmount.LOWEST,
```

Comment:
Should maintain `LOW`

Reply:
if you check the gauges config from chat it sid t change the fee on this

Reply:
this is the fee tier of the pool. Should be fixed